package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Wrapper for ActiveDirectory that gets the group for the username passed in through the x-iisnode-auth_user header",
   "main": "index.js",
   "scripts": {
+    "lint": "standard --verbose | snazzy",
```

Comment:
Could avoid the need for global install of standard/snazzy by changing the lint script like this.
```json
{
    "lint": "./node_modules/.bin/standard --verbose | ./node_modules/.bin/snazzy"
}
```